Engine/Annotations/Annotation.cs

```diff
@@ -2253,6 +2253,20 @@ public override string ToString()
             }
         }
 
+        class EnabledStringAnnotation : IStringValueAnnotation
+        {
+            readonly AnnotationCollection a;
+            public EnabledStringAnnotation(AnnotationCollection a)
+            {
+                this.a = a;
+            }
+
+            public string Value
+            {
+                get => StringConvertProvider.GetString(a.Get<IObjectValueAnnotation>().Value);
+                set => a.Get<IObjectValueAnnotation>().Value = StringConvertProvider.FromString(value, a.Get<IReflectionAnnotation>()?.ReflectionInfo, a?.ParentAnnotation?.Source ?? a.Source);
```

Comment:
Could we just use EnabledStringConvertProvider directly here instead of consulting the stack of providers? Or do we think it's possible that a plugin StringConvertProvider is needed?